shell/platform/darwin/macos/framework/Source/FlutterEngineTest.mm

```diff
@@ -219,9 +220,12 @@ @implementation MockableFlutterEngine
   latch.Wait();
 
   stdout_capture.Stop();
+  stderr_capture.Stop();
 
   // Verify hello world was written to stdout.
   EXPECT_TRUE(stdout_capture.GetOutput().find("Hello logging") != std::string::npos);
+  EXPECT_EQ(stdout_capture.GetOutput(), "Hello logging");
```

Comment:
I expect this to fail (missing an endline, etc), but I want to see what the full log is on the assert failure.